packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -458,7 +457,7 @@ export const ImageInput = ({
           onDrop={handleDrop}
         >
           <span css={hideVisually()}>{label}</span>
-          <Component src={src || previewImage} aria-hidden="true" />
+          <Component src={src || previewImage} alt="" />
```

Comment:
Previously we were doing `alt={alt || ""}`. This new implementation might be a bit less intuitive (if developers pass alt text, it won't be rendered) but it's more aligned with how we want the component to behave. In v7, we'll probably make it `aria-hidden` (depends on audit results though)